proto/cosmos/consensus/v1/tx.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
```

Comment:
is there no `genesis.proto` on purpose? Is it because those params are in the tendermint part of the genesis.json already?